twas-base/src/main/scripts/virtualimage.properties

```diff
@@ -0,0 +1,29 @@
+#      Copyright (c) Microsoft Corporation.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+# 
+#           http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+WAS_BASE_TRADITIONAL=com.ibm.websphere.BASE.v90
+IBM_JAVA_SDK=com.ibm.java.jdk.v8
+REPOSITORY_URL=https://www.ibm.com/software/repositorymanager/entitled
+WAS_ND_VERSION_ENTITLED=ND.v90_9.0.5007
```

Comment:
We should check for Base Entitlement:
WAS_BASE_VERSION_ENTITLED=BASE.v90_9.0.5009

Reply:
Good catch @git4rk , I've fixed it with commit ca51c84 based on your comment, thanks.